frontend/src/components/Card.jsx

```diff
@@ -15,15 +21,39 @@ function Card({ card }) {
     return colorClassName;
   };
   return (
-    <section className={`card ${getColorClassName()}`} key={card.id}>
-      <h1>{card.name}</h1>
-      <ul>
-        <li>EBC : {card.ebc}</li>
-        <li>ABV : {card.abv}</li>
-        <li>IBU : {card.ibu}</li>
-        <li>SRM : {card.srm}</li>
-      </ul>
-    </section>
+    <>
+      <section
+        className={`card ${getColorClassName()}`}
+        key={card.id}
+        onClick={() => {
+          changeCardSelect(card.id);
+        }}
+        onKeyDown={() => {
+          changeCardSelect(card.id);
+        }}
+        role="button"
+        tabIndex={card.id}
+      >
```

Comment:
onClick, onKeyDown, role="button", tabIndex : congratulations, you invented the button tag :)